src/Item.h

```diff
@@ -15,10 +15,10 @@ namespace tt
 
         public:
 
-            Item(       std::string id,
-                        sf::Texture texture, 
+            Item(       const std::string&  id,
```

Comment:
I lead you astray in this instance because I didn't realize `Items` was subclassed from `AnimatedSprite`. Passing the `Texture` reference here **may** not work or cause undefined behavior because you're trying to *move* a temporary reference. Then again, it might be fine. Also, it might depend on the compiler/platform. I'm not really sure. Best bet is probably to change *this particular one* back to how you had it.

And to really mess with you, the `nl::json` object probably **should** be a `const-ref`. 🤣

Reply:
I'll fix it.

Reply:
I hate C++

Reply:
C++ is awesome once you learn the quirks.